src/bento/components/bento-base-carousel/1.0/test/standalone/test-bento-base-carousel.js

```diff
@@ -0,0 +1,352 @@
+import {CSS} from '#build/bento-base-carousel-1.0.css';
+
+import {BaseElement as BentoBaseCarousel} from '#bento/components/bento-base-carousel/1.0/base-element';
+import {useStyles} from '#bento/components/bento-base-carousel/1.0/component.jss';
+import {adoptStyles} from '#bento/util/unit-helpers';
+
+import {createElementWithAttributes, waitForChildPromise} from '#core/dom';
+import {setStyles} from '#core/dom/style';
+import {mod} from '#core/math';
+import {toArray} from '#core/types/array';
+
+import {defineBentoElement} from '#preact/bento-ce';
+
+import {waitFor} from '#testing/helpers/service';
+
+describes.realWin(
+  'bento-base-carousel:1.0',
+  {
+    amp: false,
+  },
```

Comment:
I know this is copied so don't consider this blocking.

Do we still need `realWin` here? We're trying to get rid of as many of these as possible

Reply:
not sure but we do definitely need the web component tests to run chrome/puppeteer so maybe?

Reply:
we can test web comps with [jsdom](https://dev.to/ficusjs/unit-testing-web-components-with-ava-and-jsdom-2ofp) as well, but not sure how common that is

Reply:
There is an excellent chance that this DOES still need `realWin` but I wanted to draw attention to the the subject.
Both `realWin` and `fakeWin` use jsdom to provide a window object in a different way but there is a potential for tests using `realWin` to effect other tests.